users/migrations/0008_merge_20260127_0248.py

```
@@ -0,0 +1,14 @@
+# Generated by Django 5.2.7 on 2026-01-27 02:48
+
+from django.db import migrations
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('users', '0007_update_user_field_help_texts'),
+        ('users', '0007_user_is_curator_alter_user_advanceduser_and_more'),
+    ]
```

Ruff (0.14.14)

Mutable class attributes should be annotated with `typing.ClassVar`

(RUF012)